dms/controllers/portal.py

```diff
@@ -75,85 +131,64 @@ def portal_my_dms(
         if not filterby:
             filterby = "name"
         # domain
-        if not dms_directory_id:
-            domain = [
-                (
-                    "id",
-                    "in",
-                    request.env["dms.directory"]._get_own_root_directories(
-                        request.env.user.id
-                    ),
-                )
-            ]
-        else:
-            domain = [("is_hidden", "=", False), ("parent_id", "=", dms_directory_id)]
+        domain = [("is_hidden", "=", False), ("parent_id", "=", dms_directory_id)]
         # search
         if search and search_in:
             search_domain = []
             if search_in == "name":
                 search_domain = OR([search_domain, [("name", "ilike", search)]])
             domain += search_domain
         # content according to pager and archive selected
-        items = (
+        dms_directory_items = (
             request.env["dms.directory"]
-            .with_user(request.env.user.id)
+            .sudo(request.env.user.id)
```

Comment:
```suggestion
            .with_user(request.env.user.id)
```

Reply:
In V13 is needed use `with_user(id)` instead of `sudo(id)`.

Reply:
Exactly, that's why I'm requesting this change...